browser/src/control/Control.Tabs.js

```diff
@@ -354,26 +366,35 @@ L.Control.Tabs = L.Control.extend({
 		// support duplication with ctrl in the future.
 		e.dataTransfer.dropEffect = 'move';
 
-		this.classList.add('tab-dropsite');
+		var currentTabId = e.target.id;
+
+		// check if the dragged tab is over the dropZoneEndContainer
+		if (currentTabId == 'drop-zone-end-container')
```

Comment:
I think we can avoid this block of code below.
Drop zone is always before tab button.
We can use: `e.target.previousElementSibling.addClass(...)`,
no need for regex to get number, and searching in the DOM for a correct node :)